project/BuildHelper.scala

```diff
@@ -72,13 +73,19 @@ object BuildHelper {
   def stdSettings(prjName: String) = Seq(
     name := s"$prjName",
     scalacOptions := stdOptions,
-    crossScalaVersions := Seq("2.13.0", "2.12.9", "2.11.12"),
+    crossScalaVersions := Seq("2.13.0", "2.12.9", "2.11.12", dottyVersion),
     scalaVersion in ThisBuild := crossScalaVersions.value.head,
     scalacOptions := stdOptions ++ extraOptions(scalaVersion.value),
-    libraryDependencies ++= compileOnlyDeps ++ Seq(
-      compilerPlugin("org.typelevel"   %% "kind-projector"  % "0.10.3"),
-      compilerPlugin("com.github.ghik" %% "silencer-plugin" % "1.4.2")
-    ),
+    libraryDependencies ++= {
+      if (isDotty.value)
+        Seq(("com.github.ghik" % "silencer-lib_2.13.1" % "1.6.0" % Provided).withDottyCompat(scalaVersion.value))
+      else
+        Seq(
+          "com.github.ghik" %% "silencer-lib" % "1.4.2" % "provided",
```

Comment:
@LGLO You can get rid of `silencer-lib` by updating to `silencer-1.7.0` & using `scala.annotation.nowarn` (You'll need to add `scala-collection-compat` for `@nowarn` on 2.12 & 2.11)
https://github.com/ghik/silencer/releases/tag/v1.7.0

Reply:
I'm not sure if it is still needed at all, probably you can just remove it.

Reply:
Removed.
I also updated Scala versions and kind-projector version.